### PR TITLE
Add DELETE /users/:id/sessions

### PR DESCRIPTION
**Pull request by joelfranusic-okta**
For killing user sessions

@karlmcguinness-okta

### CODE REVIEW

File: _source/docs/api/resources/users.md [hunk 1]
```diff
@@ -1853,6 +1853,27 @@ curl -v -X GET \
 ]
 ~~~
 
+### Kill User Sessions
```
Comment (karlmcguinness-okta) [inline _source/docs/api/resources/users.md]: This should probably go under a new section for now as its really an admin action and not exposing a resource graph of sessions (e.g. GET or Enumerate).  

Should we create a new section such as Security Operations or just for not throw it under Credential Operations?

Comment (joelfranusic-okta) [inline _source/docs/api/resources/users.md]: A "Security Operations" section would make sense if we had a plurality of operations. For now, let's just throw it under Credential Operations.

Comment (joelfranusic-okta) [inline _source/docs/api/resources/users.md]: Actually, I changed my mind. I'm creating a "Security Operations" section.

**Comment by joelfranusic-okta** [comment]
@karlmcguinness-okta - I've updated this PR per your comments. How does this look now?

**Comment by cwu-okta** [comment]
@tthompson-okta

**Comment by mystiberry-okta** [comment]
@joelfranusic-okta just checking in--are you planning to merge this or should I be looking at it? Thanks!

**Comment by joelfranusic-okta** [comment]
@mystiberry-okta I could use your help getting this across the finish line.

**Comment by mystiberry-okta** [comment]
@karlmcguinness-okta or @tthompson-okta is this okay to merge?

**Comment by mystiberry-okta** [comment]
These changes have been already added to http://developer.okta.com/docs/api/resources/users.html#user-sessions

**Pull request closed without merging**